views/content/dao.md

```diff
@@ -128,7 +128,6 @@ The way this particular democracy works is that it has an **Owner** which works
                members[id] = Member({member: targetMember, memberSince: now, name: memberName});
             } else {
                 id = memberId[targetMember];
-                Member m = members[id];
```

Comment:
What I believe the code should be is:

```
if (memberId[targetMember] == 0) {		             
    memberId[targetMember] = members.length;		              
    id = members.length++;
} else {		             
    id = memberId[targetMember];		
}

members[id] = Member({member: targetMember, memberSince: now, name: memberName});
````

So you're right, that line must be removed and then the members[id] line must be changed..